xwiki-platform-core/xwiki-platform-user/xwiki-platform-user-profile/xwiki-platform-user-profile-ui/src/main/resources/XWiki/XWikiUserProfileSheet.xml

```diff
@@ -89,7 +89,7 @@ $services.localization.render('platform.core.profile.category.profile.disabled')
     #end
     &lt;div class='userInfo'&gt;
     #if($xcontext.action == 'view' &amp;&amp; $hasEdit)
-      &lt;div class='editProfileCategory'&gt;&lt;a href="$doc.getURL('edit', 'editor=inline&amp;amp;category=profile')"&gt;&lt;span class='hidden'&gt;$services.localization.render('platform.core.profile.category.profile.edit')&lt;/span&gt;&lt;/a&gt;&lt;/div&gt;
+      &lt;div class='editProfileCategory'&gt;&lt;a title="$services.localization.render('platform.core.profile.category.profile.edit')" href="$doc.getURL('edit', 'editor=inline&amp;amp;category=profile')"&gt;&lt;span class='hidden'&gt;$services.localization.render('platform.core.profile.category.profile.edit')&lt;/span&gt;&lt;/a&gt;&lt;/div&gt;
```

Comment:
This line needs to be formatted to not exceed 120 character limit. The title attribute value needs to be XML escaped using ``$escapetool.xml(...)``. The edit link label should probably use ``sr-only`` instead of ``hidden`` (so that screen readers read it).

Reply:
@mflorea Is It the right approach to escape xml and limiting the characters to 120?

`#set ( $editProfileCategory = $escapetool.xml($services.localization.render('platform.core.profile.category.profile.edit')))`
 
**Then** 
`title="$editProfileCategory"`

Reply:
That is an option, but not the only one (and maybe not the best). Start by formatting the HTML as it should be.